app.js

```diff
@@ -0,0 +1,56 @@
+const {
+  execTerraform,
+  parseVars,
+  makeVarFile,
+} = require("./helpers");
+
+async function executeTerraformInit(action, settings) {
+  return exeTf("init", action, settings);
+}
+
+async function executeTerraformApply(action, settings) {
+  return exeTf("apply", action, settings);
+}
+
+async function executeTerraformPlan(action, settings) {
+  return exeTf("plan", action, settings);
+}
+
+async function executeTerraformDestroy(action, settings) {
+  return exeTf("destroy", action, settings);
+}
+
+async function exeTf(mode, action, settings) {
```

Comment:
For some reason my comment about naming did not get submitted.
`exeTf` semantically is the same as `execTerraform`, yet it does something else, please name it accordingly.
Also `execTerraform` is always wrapped with `exeTf`, maybe it's worth to merge these functions then?

`mode` is passed directly to `execTerraform` where it's named `command`, so what exactly is it?

Reply:
The methods are full-sized executeTerraformInit, executeTerraformApply, etc. I left them all named as they were.
They each used to call helper function execTerraform directly. I left it named as it was. It is the function that actually executes the command.

What I decided was to consolidate all methods into one concise master method and then call execTerraform. To avoid confusion among so many things named so similarly, and because my function name appears only in one file spanning 16 lines of code, I figured according to my habits that a very terse name would do.

But if we want a long meaningful name it's probably "buildTerraformCommandLine". We should also rename the methods themselves too - "init", "plan", "apply", and "destroy", and change the helper function to "executeTerraform".

Reply:
awaiting a decision on this - either leave as is, or buildTerraformCommandLine+rename methods, or maybe you have another design.